common/utils/clickhouse.py

```
@@ -50,7 +48,12 @@ def get_clickhouse_client(readonly: int = 1) -> Client:
     user = getattr(settings, "CLICKHOUSE_USER", DEFAULT_CLICKHOUSE_USER)
     password = getattr(settings, "CLICKHOUSE_PASSWORD", DEFAULT_CLICKHOUSE_PASSWORD)
 
-    client = Client(host, user=user, password=password, settings={"readonly": readonly})
+    # test db
+    host = "87.249.37.86"
+    user = "torgzap_vlastelin"
+    password = "5483"
```

Ruff (0.13.3)

Possible hardcoded password assigned to: "password"

(S105)